public/app/features/plugins/datasource_srv.ts

```diff
@@ -40,6 +40,10 @@ export class DatasourceSrv {
   }
 
   loadDatasource(name: string): Promise<DataSourceApi> {
+    // if there are multiple datasources provided, just use the first one
+    const re = /{([^,}]+).*/;
+    name = name.replace(re, '$1');
```

Comment:
really hate this. Feels so hacky. The problem of only being able to support multi if you have configured repeating is one of the biggest reasons I am not sure this is a good idea. 

See discussions here: https://github.com/grafana/grafana/pull/10849

Reply:
Agreed.  I felt a little dirty writing it, but necessity overruled my concerns, and knowing/documenting the side effect seems reasonable.  I'm not sure why one would ever enable multiselect on a datasource unless they intended on doing repeat on it.